source/Octopus.Client/Model/LoginCommand.cs

```diff
@@ -24,5 +24,10 @@ public class LoginCommand
         /// Whether the cookie should be persistent.
         /// </summary>
         public bool RememberMe { get; set; }
+
+        /// <summary>
+        /// Encapsulated state for post successful login.
```

Comment:
I think we should mention this will be set automatically by the client when the request is made, but can be set explicitly if so desired.